cmd/config/api/api.go

```diff
@@ -83,9 +90,16 @@ func LookupConfig(kvs config.KVS) (cfg Config, err error) {
 		return cfg, err
 	}
 
+	readyDeadline, err := time.ParseDuration(env.Get(config.EnvAPIReadyDeadline, kvs.Get(apiReadyDeadline)))
+	if err != nil {
+		fmt.Println(err, kvs.Get(apiReadyDeadline))
```

Comment:
@harshavardhana it prints error here `time: invalid duration` and `kvs.Get(apiReadyDeadline)` returns empty string. Do you know what i am missing? i have added default values in `DefaultKVS`